MontyHallTests/ConsoleGameTests.cs

```diff
@@ -0,0 +1,162 @@
+using MontyHallv2.DoorCreation;
+using MontyHallv2.GameModes;
+using MontyHallv2.GameShowStaff;
+using MontyHallv2.Interfaces;
+using MontyHallv2.Messages;
+using MontyHallv2.Random;
+using Moq;
+
+namespace TestProject1.moq;
+
+public class ConsoleGameTests
+{
+    private readonly List<Door> _doors;
+    private readonly ConsoleGame _consoleGame = new();
+    private const int CarDoor = 0; 
+
+    public ConsoleGameTests()
+    {
+        //Assert
+        _doors = Enumerable.Range(1, 3)
+            .Select(_ => new Door())
+            .ToList();
+        
+        _doors[CarDoor].InjectCarToDoor();
+    }
+
+    [Fact]
+    public void GivenPlayerChooseDoorIsCalled_WhenUserIsPromptedToOpenADoor_ThenADoorWillBePicked()
+    {
+        //Arrange
+        var doorToPick = "1";
+        var stringReader = new StringReader(doorToPick);
+        Console.SetIn(stringReader);
+        const int expectedExactlyOneDoorToBePicked = 1;
+        
+        //Act
+        _consoleGame.PlayerChooseDoor(_doors);
+        var numOfDoorsPicked = _doors.FindAll(door => door.HasPlayerPicked()).Count;
+        
+        //Assert
+        Assert.Equal(expectedExactlyOneDoorToBePicked, numOfDoorsPicked);
```

Comment:
```suggestion
        
        //Act
        _consoleGame.PlayerChooseDoor(_doors);
        var numOfDoorsPicked = _doors.FindAll(door => door.HasPlayerPicked()).Count;
        
        //Assert
        Assert.Equal(1, numOfDoorsPicked);
```
maybe just me, but less is more here. alternatively, where the expected value is a bit more complex than a simple `1` `int`, perhaps `var expected = 1;` would suffice